packages/contentful/migrations/0143-create-features-page-content-type.cjs

```diff
@@ -0,0 +1,140 @@
+module.exports = function(migration) {
+  const voteableFeature = migration
+    .createContentType('voteableFeature')
+    .name('Feature')
+    .description(
+      'A feature that users can vote on.'
+    )
+    .displayField('name');
+
+  voteableFeature
+    .createField('name')
+    .name('Title')
+    .type('Symbol')
+    .localized(true)
+    .required(true)
+    .validations([{ size: { max: 100 },
+      message: 'Text must be max. 100 characters.' }])
+    .disabled(false)
+    .omitted(false);
+
+  voteableFeature
+    .createField('identifier')
```

Comment:
I'm pretty sure we can just use the contentful sys id instead of having to come up with our own ones.